lib/input/reader/amqp_1.go

```diff
@@ -15,12 +18,18 @@ import (
 	btls "github.com/Jeffail/benthos/v3/lib/util/tls"
 )
 
+const (
+	lockRenewResponseSuffix = "-response"
+	lockRenewRequestSuffix  = "-request"
+)
+
 //------------------------------------------------------------------------------
 
 // AMQP1Config contains configuration for the AMQP1 input type.
 type AMQP1Config struct {
 	URL           string      `json:"url" yaml:"url"`
 	SourceAddress string      `json:"source_address" yaml:"source_address"`
+	RenewLock     bool        `json:"renew_lock" yaml:"renew_lock"`
```

Comment:
Since this is Azure specific I'd suggest we prefix the field name to `azure_renew_lock`.